tokio/src/syscall/mod.rs

```diff
@@ -0,0 +1,50 @@
+//! The [syscall] module is intended to provide a centralized location
+//! for interacting with OS resources such as disks and network.
+//!
+//! ## Extension
+//! The Syscall trait allows hooking into implementations of Tokio
+//! disk and networking resources to supply alternate implementations
+//! or mocks.
+//!
+//! Extension requires compiling with `--cfg tokio_unstable` in addition
+//! to the `syscall` feature flag.
+//!
+//! [syscall]:crate::syscall
+
+cfg_udp! {
+    mod udp;
+    cfg_syscall! {
+        pub use udp::UdpResource;
+    }
+    cfg_not_syscall! {
+        pub(crate) use udp::UdpResource;
+    }
+}
+
+cfg_syscall! {
+    use std::{
+        fmt::Debug,
+        io, net,
+        task::{Context, Poll},
+    };
```

Comment:
Nit: elsewhere in Tokio, we prefer not to use multi-line block imports, so for consistency, this should be:
```suggestion
    use std::fmt::Debug;
    use std::{io, net};
    use std::task::{Context, Poll};
```

Reply:
Thank you!